src/epd7in5_v3/mod.rs

```diff
@@ -151,8 +158,9 @@ where
         dc: DC,
         rst: RST,
         delay: &mut DELAY,
+        delay_ms: Option<u8>,
     ) -> Result<Self, SPI::Error> {
-        let interface = DisplayInterface::new(cs, busy, dc, rst);
+        let interface = DisplayInterface::new(cs, busy, dc, rst, delay_ms);
```

Comment:
Optional:
```suggestion
        let interface = DisplayInterface::new(cs, busy, dc, rst, delay_ms.unwrap_or(DEFAULT_DELAY);
```
Where Default Delay is the old 20ms delay from below.

Reply:
the default value is handled inside interface

Reply:
I was referring the old default for this specific display which was 20ms before

Reply:
I don't think there is any value in this default. The delay tells how long to wait before looping, and this is linked to cpu usage and not to the device.